frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/grammar/MacheteLanguage.java

```diff
@@ -0,0 +1,11 @@
+package com.virtuslab.gitmachete.frontend.file.grammar;
+
+import com.intellij.lang.Language;
+
+public class MacheteLanguage extends Language {
+  public static final MacheteLanguage INSTANCE = new MacheteLanguage();
+
+  protected MacheteLanguage() {
```

Comment:
Hmm private c'tor + final class? I guess there won't be any need for extending `MacheteLanguage` (?)